### PR TITLE
First and last query guide

### PR DESCRIPTION
**Pull request by sanderson**
- [x] Tests pass (no build errors)
- [x] Rebased/mergeable

### CODE REVIEW

File: content/v2.0/query-data/flux/first-last.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
+---
+title: Query first and last values
+seotitle: Query first and last values in Flux
+list_title: First and last
+description: >
+  Use the [`first()`](/v2.0/reference/flux/stdlib/built-in/transformations/selectors/first/) or
+  [`last()`](/v2.0/reference/flux/stdlib/built-in/transformations/selectors/first/) functions
```
Comment (kelseiv) [inline content/v2.0/query-data/flux/first-last.md:7]: selectors/last (same line 28)

File: content/v2.0/query-data/flux/first-last.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+
+data
+  |> last()
+```
+
+{{% note %}}
+By default, InfluxDB returns results sorted by time, however you can use the
+[`sort()` function](/v2.0/reference/flux/stdlib/built-in/transformations/sort/)
+to change how results are sorted.
+**`first()` and `last()` respect the sorting of input tables.**
```
Comment (kelseiv) [inline content/v2.0/query-data/flux/first-last.md:45]: does "respect the sorting of...." mean these functions return results sorted by time by default, or specified sort order with sort function?

File: content/v2.0/query-data/flux/first-last.md [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+```
+
+| _time | _value |
+| ----- |:------:|
+| 0001  | 1.0    |
+{{% /flex-content %}}
+{{< /flex >}}
+
+### last
+`first()` returns the last non-null record in an input table.
```
Comment (kelseiv) [inline content/v2.0/query-data/flux/first-last.md:75]: last () returns

File: content/v2.0/query-data/flux/first-last.md [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+| 0004  | 3.0    |
+{{% /flex-content %}}
+{{< /flex >}}
+
+## Use first() or last() with aggregateWindow()
+[`aggregateWindow()`](/v2.0/reference/flux/stdlib/built-in/transformations/aggregates/aggregatewindow/)
+segments data into windows of time, aggregates data in each window into a single
+point, and then removes the time-based segmentation.
+It is primarily used to [downsample data](/v2.0/process-data/common-tasks/downsample-data/).
+`aggregateWindow` supports selector functions such as `first()` and `last()`.
```
Comment (kelseiv) [inline content/v2.0/query-data/flux/first-last.md:106]: maybe drop line 106 and slightly reword line 102:

Use first or last w aggregateWindow to segment data into....

File: content/v2.0/query-data/flux/first-last.md [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+
+{{< code-tabs-wrapper >}}
+{{% code-tabs %}}
+[first](#)
+[last](#)
+{{% /code-tabs %}}
+{{% code-tab-content %}}
+```js
+data
+  |> aggregateWindow(every: 5m, fn: first)
```
Comment (kelseiv) [inline content/v2.0/query-data/flux/first-last.md:116]: maybe show how these functions look w sample input/output data?

File: content/v2.0/query-data/flux/first-last.md (hunk 1 above)
Comment (sanderson) [reply content/v2.0/query-data/flux/first-last.md:7]: Nice catch! Thank you!

File: content/v2.0/query-data/flux/first-last.md (hunk 2 above)
Comment (sanderson) [reply content/v2.0/query-data/flux/first-last.md:45]: Updated a little bit to clarify.

File: content/v2.0/query-data/flux/first-last.md (hunk 5 above)
Comment (sanderson) [reply content/v2.0/query-data/flux/first-last.md:116]: Update the examples to show input and output.

**Pull request merged**